apps/test/unit/templates/rubrics/EvidenceLevelsTest.jsx

```diff
@@ -30,4 +31,17 @@ describe('EvidenceLevels', () => {
     // Two BodyThreeText per evidence level
     expect(wrapper.find('BodyThreeText').length).to.equal(4);
   });
+
+  it('calls radioButtonCallback when understanding is selected', () => {
+    const callback = sinon.stub();
+    const wrapper = mount(
```

Comment:
can this use shallow over mount? (it may not be able to, not sure)

Reply:
It has to be mount for the function stub to work.